index.html

```diff
@@ -2,9 +2,16 @@
 <html>
 <head>
   <title></title>
+  <script src="main.js"></script>
 </head>
 <body>
-
-  <script src="main.js"></script>
+    <form>
+        <fieldset>
+        <legend> Enter your reservation name:</legend>
+        <input id="input" name="userInput" type="text" />
+        <button type="button"  onclick="claimReservation(userInput)">submit</button>
```

Comment:
doing `claimReservation(userInput)` is ok for now but might not always work for dynamically generated content
we prefer to get the information from the input using `document.getElement...`
later in the course we'll do this with react so that won't even be necessary, but פרה פרה